CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- (internal) Rely on `Ingress` for OAuth2 proxy to configure TLS for Prometheus
+  domain, as it also configures management of the certificates, instead of
+  creating copies which could break access in case they became out of date.
```

Comment:
BTW, to me this seems like an internal change, those certs weren't used by anything else and only existed in per-cluster namespaces, so I would categorise it as an internal implementation change with no difference to functionality or exposed interfaces.

If anyone thinks I should mention removal of those copied certs, add another entry under `Removed` heading, or move this under `Removed` let me know. I'm going to merge this after lunch if there are no vetoes.

Reply:
OK, going with this 🚀